store/postgres/src/entities.rs

```diff
@@ -527,8 +491,8 @@ impl Connection {
         entity: &Entity,
     ) -> Result<usize, StoreError> {
         match &*self.metadata {
-            Storage::Json(json) => json.update_metadata(&self.conn, key, entity),
-            Storage::Relational(_) => unreachable!("relational storeage is not used for metadata"),
+            Storage::Json(_) => unreachable!("JSONB storage of subgraph metadata is not supported"),
```

Comment:
This really is unreachable: we've set up things so that we will never get here (if we do that indicates a bug somewhere) rather than something that's unimplemented, i.e., something that we should have implemented but haven't really gotten around to.

Reply:
True